src/cpp/core/system/PosixCrypto.cpp

```diff
@@ -257,6 +257,11 @@ core::Error rsaInit()
    const int KEY_SIZE = 1024;
    const int ENTROPY_BYTES = 4096;
 
+   BIGNUM *bn = BN_new();
```

Comment:
It looks like this is never freed if the first branch of the `#if OPENSSL_VERSION_NUMBER` branch is taken.